node/src/config/arguments.js

```diff
@@ -49,6 +50,14 @@ const argumentsMap = {
     prompt: 'Project ID',
     group: ['deploy', 'destroy', 'approve', 'cancel']
   },
+  [WORKSPACE_NAME]: {
+    name: WORKSPACE_NAME,
+    alias: 'w',
+    type: String,
+    description: 'Terraform Workspace name - cannot be changed after the first deployment',
+    prompt: 'Terraform Workspace Name',
```

Comment:
`prompt` value is necessary only for options that are a part of the saved configuration

Reply:
@yaronya so...?  
```suggestion
```